fineract-provider/src/main/java/org/apache/fineract/portfolio/client/service/ClientChargeReadPlatformServiceImpl.java

```diff
@@ -151,7 +151,7 @@ public Page<ClientChargeData> retrieveClientCharges(Long clientId, String status
         // filter for paid charges
         if (pendingPayment != null && pendingPayment) {
```

Comment:
I know you haven't implemented this but since you touched the other condition, can we maybe switch this to BooleanUtils.isTrue and respectively BooleanUtils.isFalse?

Much easier to read.